README.dev.md

```diff
@@ -15,8 +15,25 @@
       - Make sure to include the sha256 digest for each image, which [ensures that each image pull is immutable](https://renovatebot.com/docs/docker/#digest-pinning). Use `docker inspect --format='{{index .RepoDigests 0}}' $IMAGE` to get the digest.
 
 - Open a PR and wait for buildkite to pass and for your changes to be approved, then merge and check out master.
-- Test what is currently checked in to master by [installing](docs/install.md) Sourcegraph on fresh cluster.
-- Create a git tag and push it to the repository:
+- Test what is currently checked in to master by [installing](docs/install.md) Sourcegraph on a fresh cluster:
+  - Provision a new cluster:
+    1.  Create a cluster unique name that is identifiable to you (e.g `ggilmore-test`) in the [Sourcegraph Auxiliary GCP Project](https://console.cloud.google.com/kubernetes/list?project=sourcegraph-server&organizationId=1006954638239). You can create pool that has `3` nodes, each with `8` vCPUs and `30` GB memory (for a total of `24` vCPUs and `90` GB memory).
```

Comment:
Can we script this with something like https://github.com/sourcegraph/infrastructure/blob/master/sh/create-cluster.sh?

Reply:
It can be, but that can be solved in a separate PR. I can file an issue for it.